app/src/main/java/org/grapheneos/pdfviewer/Utils.java

```diff
@@ -39,7 +39,7 @@ public static String parseDate(String date) throws ParseException {
 
         final Calendar calendar = Calendar.getInstance();
         final int currentYear = calendar.get(Calendar.YEAR);
-        int year;
+        int year = 0;
```

Comment:
It's normal to rely on zero initialization in Java.

Reply:
Always thought local variables weren't initialised automatically. Unless that's changed.

I guess it's more of a style thing since the other date variables are initialised.

Can be removed if you want.

Reply:
For a local variable it just won't allow it to be used before it's initialized. If you want to clean this up you should just move the declaration where it's first initialized.